### DISCUSSION TITLE
Maintenance Mode and Notifications

### DISCUSSION THREAD

**Discussion by artgoldberg** [uwrit/leaf#532] [Q&A] [answered]
Hi Folks

I'm deploying [Leaf 3.11](https://github.com/uwrit/leaf/releases/tag/v3.11.0). To start, I'm testing an instance that doesn't use authentication or authorization. That is, `Authentication.Mechanism = "UNSECURED"` and `Authorization.Mechanism = "UNSECURED"`.

With these settings, both Maintenance Mode and Notifications do not work. This makes sense to me, as both these features use the user's identity. But I'm writing to check that this behavior is expected.

Thanks
Arthur

**Comment by ndobb** [comment, marked as answer]
Hi @artgoldberg - yes, that is correct and makes sense. When we developed those features I hadn't envisioned them being used in any cases where users were not authenticated, and so they are designed with that assumption. Thanks for pointing that out.

**Reply by artgoldberg** [reply]
You're welcome.
